packages/contracts-bedrock/test/scripts/L2Genesis.t.sol

```diff
@@ -0,0 +1,242 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Test } from "forge-std/Test.sol";
+import { stdJson } from "forge-std/StdJson.sol";
+import { console2 as console } from "forge-std/console2.sol";
+import { VmSafe } from "forge-std/Vm.sol";
+
+import { Artifacts } from "scripts/Artifacts.s.sol";
+import { DeployConfig, DEV_ACCOUNT_FUND_AMT } from "scripts/DeployConfig.s.sol";
+import { Executables } from "scripts/Executables.sol";
+import { Predeploys } from "src/libraries/Predeploys.sol";
+import { L2GenesisHelpers } from "scripts/libraries/L2GenesisHelpers.sol";
+import { L2Genesis } from "scripts/L2Genesis.s.sol";
+
+/// @notice Reads a `genesis-l2.json` file, parses the `alloc`s, and runs assertions
+///         against each alloc depending on whether it's a precompile, predeploy proxy,
+///         or predeploy implementation.
+contract L2Genesis_Test is Test, Artifacts {
```

Comment:
If we didn't want to make the `L2Genesis` contract methods public, i suppose that we could make the `L2GenesisTest is L2Genesis, Test {}` - this wouldn't help with the bytecode size issue tho

Reply:
While it makes sense that the `L2Genesis` methods remain `internal` (because it looks correct), I don't see downsides to making them `public` if it makes testing easier